spec/spec_helper.rb

```diff
@@ -126,7 +126,7 @@
 OmniAuth.config.mock_auth[:openid_connect] = OmniAuth::AuthHash.new(
   {
     provider: 'openid_connect',
-    uid: '12345',
+    uid: 'https://www.cilogon.org/12345',
```

Comment:
Maybe this fine, but I am seeing that Identifier db entries have the following structure for uid:
`"http://cilogon.org/serverX/users/:id"`

Reply:
You are right. This is closer to what we are seeing in prod

Reply:
@aaronskiba I will modifying these change into a rake task after your PR #923 is merged as you have the new rake task file there.